code/game/objects/items/weapons/implants/implant_explosive.dm

```diff
@@ -27,17 +27,20 @@
 /obj/item/weapon/implant/explosive/activate(cause)
 	if(!cause || !imp_in)
 		return 0
-	if(cause == "action_button" && alert(imp_in, "Are you sure you want to activate your microbomb implant? This will cause you to explode!", "Microbomb Implant Confirmation", "Yes", "No") != "Yes")
+	if(cause == "action_button" && alert(imp_in, "Are you sure you want to activate your [name]? This will cause you to explode!", "[name] Confirmation", "Yes", "No") != "Yes")
 		return 0
 	heavy = round(heavy)
 	medium = round(medium)
 	weak = round(weak)
-	imp_in << "<span class='notice'>You activate your microbomb implant.</span>"
+	imp_in << "<span class='notice'>You activate your [name].</span>"
+	var/turf/boomturf = get_turf(imp_in)
+	var/area/A = get_area(boomturf)
+	message_admins("[key_name_admin(imp_in)]<A HREF='?_src_=holder;adminmoreinfo=\ref[imp_in]'>?</A> (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[imp_in]'>FLW</A>) has activated their [name] at <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[imp_in.x];Y=[imp_in.y];Z=[imp_in.z]'>[A.name] (JMP)</a>.")
 //If the delay is short, just blow up already jeez
 	if(delay <= 7)
 		explosion(src,heavy,medium,weak,weak, flame_range = weak)
 		if(imp_in)
-			imp_in.gib()
+			imp_in.gib(no_brain = 1)
```

Comment:
Don't recommend this, personally; on the offhand chance the crew bother recovering the brain, they can actually shove it into a borg and have it fight for them, instead.